sdcm/nemesis.py

```diff
@@ -3260,21 +3259,25 @@ def decommission_post_action():
             return new_node
 
         trigger = partial(
-            self.target_node.run_nodetool, sub_cmd="decommission", warning_event_on_exception=(Exception,), retry=0,
+            self.target_node.run_nodetool, sub_cmd="decommission", timeout=180, warning_event_on_exception=(Exception,),
+            retry=0,
         )
 
+        log_follower = self.target_node.follow_system_log(patterns=["DECOMMISSIONING: unbootstrap starts"])
+
         watcher = partial(
             self._call_disrupt_func_after_expression_logged,
-            expression="DECOMMISSIONING: unbootstrap starts",
+            log_follower=log_follower,
             disrupt_func=self.reboot_node,
             disrupt_func_kwargs={"target_node": self.target_node, "hard": True, "verify_ssh": True},
             delay=0
         )
+
         ParallelObject(objects=[trigger, watcher], timeout=1200).call_objects()
         if new_node := decommission_post_action():
-            new_node.run_nodetool("rebuild")
+            new_node.run_nodetool("rebuild", timeout=300, retry=3)
```

Comment:
@fgelcer @vponomaryov, I think this change wasn't related, and isn't needed. i.e. assuming rebuild would always finish in 5min is quite off...

Reply:
how much do you think we should add here? wait forever isn't a good option neither

Reply:
waiting for ever is better option then 5min, that is failing on the release.
in general we shouldn't be setting such limits without some solid data out of previous runs and from multiple test case.

anyhow in this case, it wasn't even the intend fix/goal.

Reply:
I agree with @fgelcer , before it was not set, and now it is explicit.
@fruch which value for timeout do you propose?

Reply:
I propose to set it back to no timeout:
https://github.com/scylladb/scylla-cluster-tests/pull/5808

Assuming a timeout here without doing the work of checking/validating it with prior data is pointless and causing confusion, like it is right now during the release, @roydahan was a about to disable this nemesis cause it's similar to a different issue with have with rebuild.

you think putting a timeout here is important enough, great put it on a task/issue and we'll assign someone to do the work, slipping such a thing in PR aiming to fix something else is something we should avoid (and we even backported it, adding more confusion on older releases as well)

Reply:
Of course we need to revert it.
How did we approve it?
Rebuild will not finish in 5 minutes for most of our cases.